docs/admin/index.md

```diff
@@ -15,4 +15,56 @@ and [API](../reference/api/index.md) docs.
 For any information not strictly contained in these sections, check out our
 [Tutorials](../tutorials/index.md) and [FAQs](../tutorials/faqs.md).
 
+## What is an image, template, devcontainer, or workspace
+
+- **Image**
+
+  - A [base image](./templates/managing-templates/image-management.md) contains
+    OS-level packages, and utilities that the Coder workspace is built on. It
+    can be an [example image](https://github.com/coder/images), custom image, or
+    one from [Docker Hub](https://hub.docker.com/search). It is defined in each
+    template.
+  - Managed by: Externally to Coder.
+
+- **Template**
+
+  - [Templates](./templates/index.md) include infrastructure-level dependencies
+    for the workspace. For example, a template can include Kubernetes
+    PersistentVolumeClaims, Docker containers, or EC2 VMs.
+  - Managed by: Template administrators from within the Coder deployment.
+
+- **Startup scripts**
+
+  - Agent startup scripts apply to all users of a template. This is an
+    intentionally flexible area that template authors have at their disposal to
+    manage the "last mile" of workspace creation.
+  - Managed by: Coder template administrators.
+
+- **Workspace**
+
+  - A [workspace](../user-guides/workspace-management.md) is the environment
+    that a developer works in. Developers on a team each work from their own
+    workspace and can use
+    [multiple IDEs](../user-guides/workspace-access/index.md).
+  - Managed by: Developers
+
+- **Development containers**
+
+  - A
+    [development container](./templates/managing-templates/devcontainers/index.md)
+    is an open-source specification for defining development environments. It is
+    generally stored in VCS alongside associated source code. It can reference
+    an existing base image, or a custom Dockerfile that will be built on-demand.
+  - Managed by: Developers
```

Comment:
```suggestion
- **Development containers**

  - A
    [development container](./templates/managing-templates/devcontainers/index.md)
    is an open-source specification for defining development environments. It is
    generally stored in VCS alongside associated source code. It can reference
    an existing base image, or a custom Dockerfile that will be built on-demand.
  - Managed by: Dev Teams
```